src/main/java/petrinet/PetriNetConf.java

```diff
@@ -99,31 +120,69 @@ public PetriNetConf() {
         .forEach(transitions::add);
   }
 
-  // Getters
+  /* Getters */
```

Comment:
dejar solo eso, no poner documentacion para geters, ya estan en su nombre
```suggestion
  /* Getters */
```